src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -1,37 +1,36 @@
+   `STATIC_CONTENT_THREADS` variable| Removed in `{{ site.data.var.ct }}` v2002.1.0 | [`SCD_THREADS`]({{ site.baseurl}}/cloud/env/variables-build.html#scd_threads)
+   `DO_DEPLOY_STATIC_CONTENT` variable | Removed in `{{ site.data.var.ct }}` v2002.1.0 | [`SKIP_SCD`]({{ site.baseurl}}/cloud/env/variables-build.html#skip_scd)
+   `STATIC_CONTENT_SYMLINK` variable | Removed in `{{site.data.var.ct}}` v2002.1.0 | None. Now, the build always creates a symlink to the static content directory `pub/static`.
+   `build_options.ini` file | Removed in `{{site.data.var.ct}}` v2002.1.0 | Use the [`.magento.env.yaml`]({{ site.baseurl }}/cloud/project/magento-env-yaml.html)) file to configure environment variables to manage build and deploy actions across all your environments.<br><br>If you build a Cloud environment that includes the `build_options.ini` file, the build fails.
 
 ### CLI command changes
 
-The {{site.var.data.ct}} package no longer supports the following CLI commands:
-
-- `m2-ece-build`–use the `vendor/bin/ece-tools build` instead
-- `m2-ece-deploy`–use the `vendor/bin/ece-tools deploy` instead
-- `m2-ece-scd-dump`–use the `vendor/bin/ece-tools config:dump` instead
-
-### Removed support the `build_options.ini` file
+The {{site.data.var.ct}} package no longer supports the following CLI commands in version 2002.1.0 and later:
 
-Use the `.magento.env.yaml` file instead. If you build a Cloud environment that includes the `build_options.ini` file, the build fails.
+ Command| Replacement
+ -------- |-------
+`m2-ece-build` | `vendor/bin/ece-tools build`
+`m2-ece-deploy` | `vendor/bin/ece-tools deploy`
+`m2-ece-scd-dump` | `vendor/bin/ece-tools config:dump`
 
 ### Package changes
 
--  **New `magento/magento-cloud-patches` package**–In {{ site.data.var.ct }} v2002.22.0 and later, we moved {{ site.data.var.ece }} patches and related functionality to a separate package,`magento/magento-cloud-patches`. See [Release notes for magento/magento-cloud-patches]({{site.baseurl}}/cloud/release-notes/mcp-release-notes.md).
+-  **New magento/magento-cloud-patches package**–We moved {{ site.data.var.ece }} patches and related functionality to a separate package, `magento/magento-cloud-patches` in {{ site.data.var.ct }} version 2002.1.0. See [Release notes for magento/magento-cloud-patches]({{site.baseurl}}/cloud/release-notes/mcp-release-notes.html).
 
-- **New `magento/magento-cloud-docker` package**–In {{ site.data.var.ct }} v2002.1.0 and later, we moved Docker development functionality to a separate package, `magento-cloud-docker`. See [Release notes for magento/magento-cloud-docker]({{site.baseurl}}/cloud/release-notes/mcd-release-notes.md).
+-  **New magento/magento-cloud-docker package**–We moved Docker development functionality to a separate package in {{ site.data.var.ct }} v2002.1.0. See [Release notes for magento/magento-cloud-docker]({{ site.baseurl }}/cloud/release-notes/mcd-release-notes.html).
```

Comment:
This has a lot of BIC. These need to be documented too.

Reply:
@shiftedreality  Can you help with a list of BIC for the Docker package?